plugin/plugin_test.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2020 the Drone Authors. All rights reserved.
+// Use of this source code is governed by the Blue Oak Model License
+// that can be found in the LICENSE file.
+
+package plugin
+
+import "testing"
+
+func TestPlugin(t *testing.T) {
+	t.Skip()
```

Comment:
might be an opportunity to test some of your helper functions, eg walkPath fileExists and validateAndSetArgs,

Reply:
Already on it, thought it would be a good exercise to learn unit testing in go.